objects/finding_info.json

```diff
@@ -0,0 +1,68 @@
+{
+  "caption": "Finding Information",
+  "description": "The Finding Information object describes metadata related to a security finding generated by a security tool or system.",
+  "extends": "object",
+  "name": "finding_info",
+  "attributes": {
+    "analytic": {
+      "requirement": "recommended"
+    },
+    "attacks":{
+      "description": "The <a target='_blank' href='https://attack.mitre.org'>MITRE ATT&CK®</a> technique and associated tactics related to the finding.",
+      "requirement": "optional"
+    },
+    "created_time": {
+      "description": "The time when the finding was created.",
+      "requirement": "optional"
+    },
+    "data_sources": {
+      "requirement": "optional"
+    },
+    "desc": {
+      "description": "The description of the reported finding.",
+      "requirement": "optional"
+    },
+    "first_seen_time": {
+      "description": "The time when the finding was first observed. e.g. The time when a vulnerability was first observed. <p>It can differ from the <code>created_time</code> timestamp, which reflects the time this finding was created.</p>",
+      "requirement": "optional"
+    },
+    "kill_chain": {
+      "requirement": "optional"
+    },
+    "last_seen_time": {
+      "description": "The time when the finding was most recently observed. e.g. The time when a vulnerability was most recently observed. <p>It can differ from the <code>modified_time</code> timestamp, which reflects the time this finding was last modified.</p>",
+      "requirement": "optional"
+    },
+    "modified_time": {
```

Comment:
Do we want `created_time_dt` `first_seen_time_dt` `last_seen_time_dt` or `modified_time_dt`?

Reply:
I don't see those in the public [Detection Findings class](https://schema.ocsf.io/1.3.0-dev/classes/detection_finding?extensions=). Where are you seeing those?

Reply:
Those `_dt` attributes get automatically applied classes when the `Date/Time` profile is enabled. They don't need to be added to the class, since this is normal behavior for that profile.